src/classes/Gaussian2D.ts

```diff
@@ -0,0 +1,219 @@
+import { GAUSSIAN_EXP_FACTOR } from '../util/constants';
+import erfinv from '../util/erfinv';
+
+import { widthToFWHM, fwhmToWidth } from './Gaussian';
+
+export { widthToFWHM, fwhmToWidth } from './Gaussian';
+
+export interface xyNumber {
+  [index: string]: number;
```

Comment:
Why this line is there ?

Reply:
it is needed when you point to properties inside of the object.

Reply:
can you show what breaks if you remove this line?